set.mm

```diff
@@ -541306,31 +541343,12 @@ extends this idea to multiple additions (TODO).
        set.  Our hypothesis here expresses tentatively it might not hold.  We
        can simplify the antecedent then, to the point where we do not need
        equation any more.  This suggests what a decent characterization of the
-       empty domain could be.  (Contributed by Wolf Lammen, 12-Mar-2023.) $)
+       empty logical domain could be.  (Contributed by Wolf Lammen,
```

Comment:
```suggestion
       empty domain of discourse could be.  (Contributed by Wolf Lammen,
```
In your mathbox, so as you prefer, but the standard term is "domain of discourse" (or "domain" for short), or sometimes "universe of discourse".